students/nartowska_karina/lesson_06_dicts_tuples_sets_args_kwargs/list_to_dictionary_function_for_fantasy_game_inventory.py

```diff
@@ -0,0 +1,22 @@
+from fantasy_game_inventory import display_inventory
+
+
+def add_to_inventory(inventory, added_items):
+    for i in added_items:
+        if i in inventory:
+            inventory[i] += 1
+        else:
+            inventory[i] = 1
```

Comment:
Here you could use inventory.setdefault(i, 1) as it was described in the lesson. Your solution is good as well :)